.travis.yml

```diff
@@ -1,17 +1,9 @@
 language: erlang
 otp_release:
-  - 17.1
   - 17.0
   - R16B03-1
-  - R16B03
```

Comment:
No, I'm dropping because `erlydtl` can't run on a lot of them.

I'm hoping to get erlydtl removed from relx and rebar3. Someone is working on just that, I'll have to check on his progress.